resources/js/store/modules/teams/actions.js

```diff
@@ -27,13 +27,59 @@ export const actions = {
                 });
 
                 context.commit('usersActiveTeam', payload);
+                context.commit('usersTeam', response.data.team);
             }
         })
         .catch(error => {
             console.error('change_active_team', error);
         });
     },
 
+    /**
+     * Leave the team
+     */
+    async LEAVE_TEAM (context, payload)
+    {
+        await axios.post('/teams/leave', {
+            teamId: payload
+        })
+        .then(response => {
```

Comment:
after every request, please add a log with the name of the action in lowercase. I find this makes debugging easy to find out what requests are being made and what actions are being called